quickstarts/microsoft.network/internal-loadbalancer-create/azuredeploy.parameters.json

```diff
@@ -5,8 +5,20 @@
     "vnetAddressPrefix": {
       "value": "10.0.0.0/16"
     },
-    "subnetPrefix": {
+    "vNetSubnetAddressPrefix": {
       "value": "10.0.0.0/24"
+    },
+    "vNetBastionSubnetAddressPrefix": {
+      "value": "10.0.2.0/24"
+    },
+    "lbPublicIPAddress":{ 
+      "value": "10.0.0.6"
+    },
+    "adminUsername": {
+      "value": "GEN-UNIQUE"
+     },
+    "adminPassword": {
+      "value": "GEN-UNIQUE"
```

Comment:
Please use `GEN-PASSWORD` instead of `GEN-UNIQUE`.

Reply:
Updated. Thanks!